kernels/volk/volk_8u_conv_k7_r2puppet_8u.h

```diff
@@ -175,6 +175,88 @@ static inline void volk_8u_conv_k7_r2puppet_8u_spiral(unsigned char* syms,
 #endif /*LV_HAVE_SSE3*/
 
 
+#if LV_HAVE_NEON
+
+#include "volk/sse2neon.h"
+
+static inline void volk_8u_conv_k7_r2puppet_8u_spiral(unsigned char* syms,
```

Comment:
Everytime I looked at the test output, I was confused. The name `spiral` is so tightly coupled with x86. I suggest to rename this implementation to `spiralneon` or `neonspiral`. Same with the puppet. It makes the intent and the way it works more obvious.

Reply:
Done that as well. Though Spiral actually comes from https://spiral.ece.cmu.edu/vitgen/. Only supports x86, but it's not something to do especially with x86 otherwise, outside of the Spiral software focusing mainly on that arch.